content/blog/entries/science-museum-implementation/contents.lr

```diff
@@ -0,0 +1,115 @@
+title: Science Museum provider implementation
+---
+categories: 
+
+cc-catalog 
+gsoc
+gsoc-2020
+---
+author: srinidhi
+---
+series: gsoc-2020-cccatalog
+---
+pub_date: 2020-05-29
+---
+body:
+Science museum is a provider with around 80,000 CC license images, currently Science museum data is ingested from Common Crawl. 
+The problem with Common Crawl is that we don't have control over the content available and this results in poor data quality.
+Science museum is one such provider that has poor data quality and there is need to improve it. So this is done by moving Science museum to an API based crawl. 
```

Comment:
Please give more intro to an "API Crawl".  I suggest a quick intro of both (API vs Common Crawl) ingestion methods and a general comparison between them.